src/endpoint/sts/ops/sts_post_assume_role.js

```diff
@@ -35,12 +35,13 @@ async function assume_role(req) {
                 Credentials: {
                     AccessKeyId: access_keys.access_key.unwrap(),
                     SecretAccessKey: access_keys.secret_key.unwrap(),
-                    Expiration: expiry,
+                    Expiration: Date.now() + expiry,
```

Comment:
How will the expiration look like? I mean will we see the milli seconds? (we should not as I understand it for example: `<Expiration>2019-11-09T13:34:41Z</Expiration>`)

Reply:
I think there is a difference - this is the API of "AssumeRole" and you send me the "GetSessionToken".
- GetSessionToken - from the output (with milliseconds): <Expiration>2011-07-11T19:55:29.611Z</Expiration>
- but in AssumeRole - from the output (without milliseconds): <Expiration>2019-11-09T13:34:41Z</Expiration> (it is part of the data inside the Credentials data type).